asyncify.mjs

```diff
@@ -48,10 +36,23 @@ function proxyGet(obj, transform) {
   });
 }
 
-class Asyncify {
-  constructor() {
+export class Asyncify {
+  constructor(options) {
```

Comment:
Can you use default parameters instead like

```suggestion
  constructor({ dataBegin, dataEnd } = {}) {
```

Using `||` like in the current patch is problematic if someone wants to actually put data at `0` address (why not, after all, it's valid in Wasm). Plus, default params make it slightly more discoverable.